jupyterhub/templates/scheduling/user-scheduler/configmap.yaml

```diff
@@ -28,7 +28,7 @@ data:
        and configuration (v1beta1) of kube-scheduler.
   */}}
   config.yaml: |
-    {{- if ge (atoi .Capabilities.KubeVersion.Minor) 21 }}
+    {{- if ge (atoi ( .Capabilities.KubeVersion.Minor | replace "+" "" )) 21 }}
```

Comment:
```suggestion
    {{- /*
      FIXME: We have added a workaround for EKS where
            .Capabilities.KubeVersion.Minor can return a
            string like "22+" instead of just "22".

            See https://github.com/aws/eks-distro/issues/1128.
    */}}
    {{- if ge (atoi (.Capabilities.KubeVersion.Minor | replace "+" "")) 21 }}
```